src/pages/Committees.js

```diff
@@ -587,4 +641,204 @@ function SportsCommittee() {
+            </ul>
+        </p>
+    )
+}
+
+function MediaElectionCommittee () {
+    return (
+        <p>
+            The Election Committee (Swedish: Valberedningen) prepares and manages the elections for certain positions in the Media chatper.
+            The committee receives nominations and candidacies, asses candidates with an interview, and make a review of the candidates for
+            the next chapter meeting. The Election Committee works to keep elections democratic and ensure all candidates have an equal treatment
+            and process.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: val@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MediaStudyBoard() {
+    return (
+        <p>
+            The Study Board (Swedish: Studienämnden) verifies that every student has a well-functioning education and safe
+            study environment. This applies at both programme-level and course-level. If you have any problem with courses and
+            programs, or have questions about them, you can contact the Study Board! Joining the committee lets you help everyone
+            have a better education and you will have a better understanding of education at KTH.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: sno@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function CommunicationCommittee () {
+    return (
+        <p>
+            The Communication Committee (Swedish: Kommunikationsnämnden) is the committee in charge of Media's external branding,
+            e.g. they update Media's Instagram, Facebook, etc. They also take care of the communications within the chapter and
+            announce events. Finally, they run a podcast on Spotify!
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: kommunikator@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Branschdag () {
+    return (
+        <p>
+            Searching for a job after KTH? Media's carrier fair (Swedish: Branschdag) is Media's fair lets you find jobs,
+            degree project or simply to enrich your network, it's up to you! Don't miss it!
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: branschdag@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function BusinessGroup () {
+    return (
+        <p>
+            The Business Relations Group (Swedish: Näringslivsgruppen) initiate contacts with companies and provide students with opportunities
+            to build a strong network of business contacts. Working for the Business Group makes it doubly as effective for you. Members
+            organise events such as mingles or recruitement events.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: nlg@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Festmästeriet () {
```

Comment:
"Festmästeriet" should be in definite form, i.e. not "Festmästeri"

Here is my suggestion for the decription:
"Party people, here is your stuff! The Party Committee (Swedish: Festmästeriet) is specialized in organizing and hosting _gasques_; traditional swedish student parties featuring a three course meal, fun activities and an afterparty open for everyone. Members of the committee can engage in various tasks, such as deciding the menu, doing PR (i.e. marketing), inventing fun competitions, or decorating the venue. Join them to maximise the fun of Media's parties."

Could you also add our Instagram: @festmasteriet_medieteknik